apis/nucleus/src/viz.js

```diff
@@ -109,6 +112,24 @@ export default function viz({ model, halo, initialError, onDestroy = async () =>
       }
       return propertyTree;
     },
+    /**
+     * Listens to custom events from inside the visualization. See useEmitter
+     * @experimental
+     * @param {string} e Event name to listen to
+     * @param {Function} cb Callback function to invoke
+     */
+    on(e, cb) {
+      emitter.on(e, cb);
+    },
```

Comment:
consider longer names for better readability
(node docs use eventName and listener (https://nodejs.org/api/events.html#emitteroneventname-listener)